src/main/java/iitema/gypsypokemon/elements/ProjectileInterface.java

```diff
@@ -4,18 +4,22 @@
  * A projectile which can be shot
  */
 public interface ProjectileInterface {
+    /**
+     * Color of projectile
+     */
+    enum CartridgeColor {YELLOW, BLUE}
```

Comment:
I think that makes it less clean, but the main reason i removed it is that it caused commit warnings.